### PR TITLE
Jaseci Docs

### PR DESCRIPTION
**Pull request by RolexAlexander**
Good Afternoon, Dr Mars, Dr Yiping and the development team. I proceeded to arrange the content in the jaseci docs as outlined in the google docs I prepared. Additionally, I also added some content to some of the sections. Furthermore, I require some information for some of the new and old sections of the docs.

### CODE REVIEW

File: docs/docs/SampleProjects_templates_and_tutorials/cononi_cai.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Cononi Cai
+
+In this section, we will provide a comprehensive guide on developing a conversational AI using the Cononi Cai Structure. Our tutorial will walk you through the process of creating an automated FAQ answer chatbot with multi-turn dialog capabilities, along with a dialog system. Additionally, we will guide you through the process of preparing your dataset for training the bi-encoder and the TFM-NER, which are used for intent classification and entity extraction, respectively. With our step-by-step instructions, you can dive right in and start building your very own conversational AI. So let's get started on this exciting journey!
```
Comment (TharukaCkasthuri) [inline docs/docs/SampleProjects_templates_and_tutorials/cononi_cai.md]: CanoniCAI

**Review by TharukaCkasthuri — COMMENTED** [review]
Better to remove empty md files.

File: docs/docs/jaseci_architype/referencing_dereferencing.md [hunk 2]
```diff
@@ -35,4 +35,10 @@ info: {
     "j_timestamp": "2022-08-10T15:57:00.577287", 
     "j_type": "node"
 }
-```
\ No newline at end of file
+```
+
+The code snippet creates a simple node with an attribute named name. It then initializes a walker named ref_deref, which creates three new nodes named node0, node1, and node2 and assigns them to the simple node.
+
+The code then assigns the reference of the first node node0 to the var variable using the & operator. The std.out() function is used to print out the reference, object, and information related to the var variable.
```
Comment (TharukaCkasthuri) [inline docs/docs/jaseci_architype/referencing_dereferencing.md]: use backtick keys to highlight lines of codes inside paragraphs. like this `std.out()`

File: jaseci_studio/README.md [hunk 3]
```diff
@@ -1,3 +1,13 @@
+# Installing Jaseci Studios
```
Comment (TharukaCkasthuri) [inline jaseci_studio/README.md]: Jaseci Studio

File: jaseci_studio/architype.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+# Archetype
+
+Archetypes are a set of abstractions offered by Jaseci, and Jaseci Studios has a powerful feature that will surprise all programmers. Jaseci Studios allows programmers to view and edit all the code related to a particular archetype, whether it be a node, edge, graph, or walker. This greatly enhances development efficiency. You are allowed to create new archetypes and write the code for them, and register them to the Jaseci application, which is amazing.
```
Comment (TharukaCkasthuri) [inline jaseci_studio/architype.md]: Jaseci Studio

**Comment by RolexAlexander** [comment]
Will make the changes. Thanks for the feedback.

File: docs/docs/SampleProjects_templates_and_tutorials/cononi_cai.md (hunk 1 above)
Comment (RolexAlexander) [reply docs/docs/SampleProjects_templates_and_tutorials/cononi_cai.md]: Good Morning, I changed the Cononi Cai to Canoni CAI and I also did the same for the others, which were cononi vision and cononi analytics. Are there any other changes needed?

File: docs/docs/jaseci_architype/referencing_dereferencing.md (hunk 2 above)
Comment (RolexAlexander) [reply docs/docs/jaseci_architype/referencing_dereferencing.md]: Thanks for this recommendation, I actually went through the docs and noticed that there are multiple occurrences of this.

File: jaseci_studio/architype.md (hunk 4 above)
Comment (RolexAlexander) [reply jaseci_studio/architype.md]: Thanks

**Review by ypkang — APPROVED** [review]

**Pull request merged**